selfdrive/car/hyundai/values.py

```diff
@@ -48,6 +48,7 @@ class HyundaiFlags(IntFlag):
   CANFD_HDA2 = 1
   CANFD_ALT_BUTTONS = 2
   CANFD_ALT_GEARS = 4
+  CANFD_CAMERA_SCC = 8
```

Comment:
We can combine in the future, for now we check CAMERA_SCC_CAR in a few places where it's difficult to add the CANFD cars (like that expLong is available).